zcbor/zcbor.py

```diff
@@ -316,7 +316,7 @@ def generate_base_name(self):
             or (next((key for key, value in self.my_control_groups.items() if value == self), None))
             or ("_" + self.value if self.type == "OTHER" else None)
             or ("_" + self.value[0].get_base_name()
-                if self.type in ["LIST", "GROUP"] and self.value is not None else None)
+                if self.type in ["LIST", "GROUP"] and self.value else None)
```

Comment:
Is the commit message a bit misleading? This is a bugfix right? To me it sounds like we are changing the intended semantics of this line.

This is a nit, but still, trying to understand the functionality.